client/app/dedicatedCloud/dashboard/main/tiles/options/order/steps/smsActivation/component.js

```diff
@@ -0,0 +1,13 @@
+import controller from './controller';
+import template from './template.html';
+
+export default {
+  bindings: {
+    usersWhoCanReceiveSMS: '<',
+  },
+  controller,
+  require: {
+    stepper: '^stepper',
```

Comment:
This is a good remark but the steps are entwined into how the stepper works so they can't be extracted

I've done this because this is how ui frameworks do it:
- either they are aware that they are in a stepper
- either each step is not a component but a part of the stepper

I've done the first solution 🎉